package.json

```diff
@@ -26,6 +26,7 @@
     "rxjs": "~7.8.1",
     "tslib": "^2.6.1",
     "zingchart": "latest",
+    "zingchart-angular": "^2.0.0",
```

Comment:
`package.json`

You should first build the library **zingchart-angular** before building the app **zing-app**. 
Because in this version, you should manually import `ZingchartAngular from "zingchart-angular/zingchart";`.

This is the way the library should work because the application will be more performant this way.
If you do not have to import ZingchartAngular, the namespace will be imported all over the application, even where it is not needed. By manually importing ZingchartAngular, the namespace will be only imported where it is actually used.